web-console/future-openshift/custom-catalogs/custom-catalogs.md

```diff
@@ -0,0 +1,69 @@
+# Adding custom catalogs to OperatorHub
+
+Admins can create custom catalogs in order to give admin console users the ability to add operators from other sources, beyond the default OperatorHub catalog sources.
+
+## Global Configuration
+
+OperatorHub sources are found through: Administation > cluster settings > global configuration > OperatorHub / sources tab.
+
+![global config](img/operatorhub-custom-catalog1.png)
+
+![OperatorHub config](img/operatorhub-custom-catalog2.png)
+
+## Sources
+
+In the OperatorHub configuration, users can see the current default sources that are feeding into OperatorHub. These sources populate OperatorHub and keep installed operators updated.
+
+![OperatorHub sources default](img/operatorhub-custom-catalog3.png)
+
+If users "Create Catalog Source" they will see a form to fill out details.
+
+![OperatorHub catalog source form](img/operatorhub-custom-catalog4.png)
+
+After the form is filled out and users select "create", they will see the newly created catalog source in the list.
+
+![OperatorHub catalog source in list](img/operatorhub-custom-catalog5.png)
+
+Once a new catalog source is created, admin users can see the operators from the catalog source from within OperatorHub.
+
+![OperatorHub with custom catalogs](img/operatorhub-custom-catalog6.png)
+
+
+## Actions
+
+Custom catalog sources can be edited, deleted, or disabled from the kebab menu in the list.
+
+![edit catalog source](img/operatorhub-custom-catalog7.png)
+
+If users delete a custom catalog source, they must first confirm the deletion by typing "delete". The deletion confirmation step will also inform users that if a source is deleted, any operators already installed from the source will no longer receive updates.
```

Comment:
Having delete in this modal is kind of tough, as I don't see the context anymore.  IMO, I think it's better to include the name of the thing that's being deleted.

Reply:
@serenamarie125 - Do you have any thoughts on what we should do in the case of bulk delete? @bmignano has been using this pattern so we were wondering if it would make sense to be consistent or not.

Reply:
@lizsurette I think using "delete" for bulk makes sense.  IMO, when doing deletion which is destructive enough to ask for confirmation + user input, showing context of what's being deleted, and making sure the user enters the actual name ( thinks rather than reacts ) makes sense.  This pattern is also used in Project & Application deletion on the dev side.